datasets/consumer-finance-complaints/consumerComplaints.py

```diff
@@ -0,0 +1,240 @@
+import csv
+import textwrap
```

Comment:
I think an import is missing here:
```suggestion
import os
import textwrap
```

Reply:
Can you also rename this file `consumer-finance-complaints.py` please ? It must have the same name as the directory